.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig helps maintain consistent coding styles across editors and IDEs
+# More details at https://editorconfig.org
```

Comment:
Interesting. Eventually I'd like ci-mgmt to ship some commit hooks to apply consistent linting and formatting for us.

Reply:
That would be great. I love editorconfig. It allows, it allows vscode to just know what to use automagically. We could ship it via ci-mgmt, like we do go-release.

Reply:
I added it because the default to .ts is 4spaces in my vscode and it kept refactoring files.